compiler/src/dotty/tools/dotc/quoted/TreeCleaner.scala

```diff
@@ -17,12 +17,14 @@ class TreeCleaner extends tpd.TreeMap {
   /** List of symbols and their types for type aliases `type T = U` */
   private[this] var aliasesSyms: List[Symbol] = Nil
   private[this] var aliasesTypes: List[Type] = Nil
+  private[this] val aliases = newMutableSymbolMap[Tree]
```

Comment:
The changes in this file are not relevant to the fix right? BTW can we add a comment for the `aliases` var just for symmetry? 😛